e2e/playwright/info-drawer/src/tests/file-preview.spec.ts

```diff
@@ -55,11 +46,11 @@ test.describe('File preview', () => {
   });
 
   test.beforeEach(async ({ loginPage }) => {
-    try {
-      await loginPage.loginUser({ username, password: username }, { withNavigation: true, waitForLoading: true });
-    } catch (error) {
-      console.error(`Main beforeEach failed: ${error}`);
-    }
+    await Utils.tryLoginUser(loginPage, username, username, 'beforeEach failed');
+  });
+
+  test.afterAll(async () => {
+    await Utils.deleteNodesSitesEmptyTrashcan(nodesApi, trashcanApi, 'afterAll failed', undefined, undefined, undefined);
```

Comment:
instead of passing 3 undefined you can pass nothing as these parameters are optional